python/modules/task.py

```diff
@@ -253,33 +238,68 @@ def __init__(self, parameters_by_value, parameters_by_name, updates_by_value, up
         self.config_dict = args['config_dict']
 
     def execute_specify(self, dict_local):
-        dir_list, folders_number = self.parsing_elements_list(dict_local)
+        elements_list, ele_number = self.parsing_elements_list(dict_local)
         processes_number = int(dict_local["number_of_cores"])
         sleep_time = int(dict_local["sleep_time"])
-        new_dirs = dir_list
+        new_elements = elements_list

```

Comment:
@stork119 No, its not. Elements_list are all collected elements in given loop, while new_elements is a list containing only those which wasn't present before. They're the same only during the first iteration.